sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlobBaseAPITest.groovy

```diff
@@ -287,6 +287,41 @@ class BlobBaseAPITest extends APISpec {
         1000      || _
     }
 
+    @Unroll
+    def "Query Input parquet"() {
+        setup:
+        String fileName = "parquet.parquet"
+        ClassLoader classLoader = getClass().getClassLoader()
+        File f = new File(classLoader.getResource(fileName).getFile())
```

Comment:
Will this work cross platform?  Loading files on Windows vs. Mac and Linux has tripped me up a lot in the past.

Reply:
Yeah this works cross platform